spec/lib/fca/import_spec.rb

```diff
@@ -51,4 +51,18 @@
       end
     end
   end
+
+  describe '#files_to_be_imported' do
+    subject(:files_to_be_imported) do
+      described_class.new(files, {}, []).files_to_be_imported
+    end
+
+    it 'returns regexp of the three files' do
```

Comment:
Why no rewrite this like as: 

```ruby
     it 'returns regexp of the three files' do
        expect(files_to_be_imported).to eq([
          /^firms_master_list2\d+\.ext$/,
          /^indiv_apprvd2\d+\.ext$/,
          /^firm_names2\d+\.ext$/
        ])
    end
```

It's a matter of style but I believe the above is easier on the eye for readability.

Reply:
To be honest I'd be ok with either style.

I'd maybe favour @Tr4pSt3R myself, but not really for readability; in general the first approach is less likely to run into line-length issues.

Reply:
@Tr4pSt3R  I choose yours, but Travis failed [here](https://travis-ci.org/moneyadviceservice/rad/builds/335732528). Then I ran rubocop with the flag -a and rubocop autocorrected to that.

Reply:
🤦‍♀️

Reply:
Hmm, Oh that's very interesting! Well if Rubocop and Travis are in agreement then I am in agreement too. 👍